tests/deprecated_io_test.py

```diff
@@ -0,0 +1,621 @@
+import glob
+import os
+import shutil
+import tempfile
+import pytest
+
+import h5py
+import numpy as np
+import pandas as pd
+from base_test import ArkoudaTest
+from context import arkouda as ak
+
+from arkouda import io_util, io
+
+TYPES = ("int64", "uint64", "bool", "float64", "str")
+SIZE = 100
+NUMFILES = 5
+verbose = True
+
+
+class DeprecatedIOTest(ArkoudaTest):
```

Comment:
Just an idea - Should we add checks to ensure this functionality throws deprecation warnings?

Reply:
Not a bad idea, I will set that up.

Reply:
Added checks that deprecated functions through deprecation warnings.